.github/workflows/build-skip.yml

```diff
@@ -0,0 +1,17 @@
+name: Build SimApp
+# This workflow allows to skip the build step if the PR does not contain any changes to the code
```

Comment:
Can you add a link to the docs

Reply:
Where in the docs?

Reply:
sorry meant this  https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/troubleshooting-required-status-checks#handling-skipped-but-required-checks in the *.yml file so people can reference the docs about why its there